participants/Filecoin-Crawl.md

```diff
@@ -0,0 +1,63 @@
+# <Filecoin-Crawl>
+
+Before submitting a PR containing your team's project information:
+-Make sure you rename this file to contain your project's name
+-Respond to add your response in each section below
+
+##(0) Project Name and Team Slack Handles
+
+Our project name is Filecoin-Crawl, We'are Filecoin-Crawl.Our team is one of filecoin miner and also we're developing IPFS applications
```

Comment:
Can you add slack emails or contact information?

Reply:
Slack Handles: Patrick-li, grandhelmsman;
Emails: desupport@grandhelmsman.com.